_about/breaking-changes.md

```diff
@@ -43,6 +43,11 @@ A Lucene upgrade forced OpenSearch to drop support for JDK 8. As a consequence,
 
 OpenSearch 2.5 contains a bug fix that corrects the behavior of the `case_insensitive` parameter for the `wildcard` query on text fields. As a result, a wildcard query on text fields that ignored case sensitivity and erroneously returned results prior to the bug fix will not return the same results. For more information, see issue [#8711](https://github.com/opensearch-project/OpenSearch/issues/8711).
 
+## 2.19.0
+
+### Nested Value Support in TextEmbeddingProcessor
+OpenSearch no longer support using Set Processor to replace the nested value like `title_tmp:_ingest._value.title_embedding` where `_ingest._value` can be replaced by Set Processor. Users need to directly specify the nested key like this `books.title:title_embedding` to get desired output. For more information, see issue [#1243](https://github.com/opensearch-project/neural-search/issues/1243)
```

Comment:
What is a "set processor"? Is this referring to a Java class? If so, user documentation should not mention it. As I understand, this is applicable to the `text_embedding` processor. How about:
```suggestion
The `text_embedding` processor no longer replaces nested values like `_ingest._value` when evaluating fields like `title_tmp:_ingest._value.title_embedding`. Instead, you must directly specify the nested key as `books.title:title_embedding` to achieve the desired output. For more information, see issue [#1243](https://github.com/opensearch-project/neural-search/issues/1243).
```